frontend/src/metabase/collections/components/CollectionCardVisualization/CollectionCardVisualization.jsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import PropTypes from "prop-types";
+import _ from "underscore";
+
+import Questions from "metabase/entities/questions";
+import Question from "metabase-lib/lib/Question";
+import Visualization, {
+  ERROR_MESSAGE_GENERIC,
+  ERROR_MESSAGE_PERMISSION,
+} from "metabase/visualizations/components/Visualization";
+import QuestionResultLoader from "metabase/containers/QuestionResultLoader";
+import LoadingAndErrorWrapper from "metabase/components/LoadingAndErrorWrapper";
+
+import { ItemLink } from "../PinnedItemCard/PinnedItemCard.styled";
+import { HoverMenu, VizCard } from "./CollectionCardVisualization.styled";
+
+const propTypes = {
+  item: PropTypes.object.isRequired,
+  collection: PropTypes.object.isRequired,
+  metadata: PropTypes.object.isRequired,
+  onCopy: PropTypes.func.isRequired,
+  onMove: PropTypes.func.isRequired,
+};
+
+function CollectionCardVisualization({
+  item,
+  collection,
+  metadata,
+  onCopy,
+  onMove,
+}) {
+  return (
+    <ItemLink to={item.getUrl()}>
+      <VizCard flat>
+        <HoverMenu
+          item={item}
+          collection={collection}
+          onCopy={onCopy}
+          onMove={onMove}
+        />
+        <Questions.Loader id={item.id}>
+          {({ question: card }) => {
+            const question = new Question(card, metadata);
+            return (
+              <QuestionResultLoader question={question}>
+                {({ loading, error, reload, rawSeries, results, result }) => {
+                  const shouldShowLoader = loading && results == null;
+                  const { errorMessage, errorIcon } = getErrorProps(
+                    error,
+                    result,
+                  );
+
+                  return (
+                    <LoadingAndErrorWrapper
+                      loading={shouldShowLoader}
+                      noWrapper
+                    >
+                      <Visualization
+                        onChangeCardAndRun={_.noop}
```

Comment:
`_.noop` instead of null/undefined here because it causes some style changes.

this how chunk of code is mostly taken from `enterprise/frontend/src/metabase-enterprise/audit_app/containers/QuestionLoadAndDisplay.jsx` with a few tweaks. This seems like the best way to do it? Main annoyance is the need for `LoadingAndErrorWrapper` because the appearance of the `Visualization` loading wrapper is different from the `Questions.Loader` wrapper